src/hb-common.cc

```diff
@@ -60,12 +60,12 @@ _hb_options_init (void)
 
 /**
  * hb_tag_from_string:
- * @str: (array length=len) (element-type uint8_t): 
- * @len: 
```

Comment:
Why all the whitespace noise? Or do you have an editor that automatically does these?

Reply:
Yes, that is the case with the our current `.editorconfig` which requests it.